src/CMakeLists.txt

```diff
@@ -962,9 +962,11 @@ list(
   fm/cp_cfm_diag.F
   fm/cp_cfm_basic_linalg.F
   fm/cp_cfm_types.F
+  fm/cp_dlaf_utils.F
```

Comment:
exclude it from the build if dlaf is disabled.

Reply:
I agree in principle, but this is not what it is done with the other packages. Do you prefer to go down a different path or be consistent with other libraries? I tried to do the latter.

Reply:
DLAF-related files added conditionally.